suppliers/src/API/EMBC.Suppliers.API/ConfigurationModule/Controllers/ConfigController.cs

```diff
@@ -113,27 +111,48 @@ public async Task<ActionResult<Config>> GetConfig()
         /// <returns>True/False, based on CRM connectivity</returns>
         private async Task<bool> CRMHealthcheck()
         {
-            bool siteDown = true;
             try
             {
+                // Simple query to CRM to make sure connection can get response
                 CRMGetListResult<SupportEntity> list = await api.GetList<SupportEntity>("era_supports", new CRMGetListOptions
                 {
                     Select = new[] { "era_name", "era_supportid" }
                 });
 
-                if (list.List.Count > 0)
-                {
-                    siteDown = false;
-                }
-                return siteDown;
+                // If query returns list with items in it, site is healthy
+                return list.List.Count > 0;
             }
             catch (Exception e)
             {
                 Console.WriteLine(e);
```

Comment:
we have a logging framework worth using

Reply:
@josephweinkamgov Can you look into the logging system being used, and use it if Dynamics can't be reached?
Also please look into the other adjustment Yossi recommended above. Up to you if that warrants a separate task, or if you want to reuse the original Dynamics Healthcheck one.